app/src/main/java/org/projectbuendia/client/models/ConceptUuids.java

```diff
@@ -25,6 +26,132 @@
     public static final String HYDRATION_UUID = "162653AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
```

Comment:
Are any of these unused now? If so, please clean them up. You can check in Android Studio.